apps/a11y-tests/src/tests/ComponentExamples.test.tsx

```diff
@@ -0,0 +1,58 @@
+import * as React from 'react';
+import * as fs from 'fs';
+import * as path from 'path';
+import * as glob from 'glob';
+import { getSarifReport } from '../getSarifReport';
+import { SarifLog } from 'axe-sarif-converter/dist/sarif/sarif-log';
+import { Result } from 'axe-sarif-converter/dist/sarif/sarif-2.0.0';
+
+// Keep only errors to reduce snapshot size
+function dehydrateSarifReport(report: SarifLog): Result[] {
+  return report.runs[0]!.results!.filter(item => item.level === 'error');
+}
+
+/* tslint:disable-next-line:no-any */
+async function testComponent(component: { name: string; pageName: string; elem: React.ReactElement<any> }) {
+  it(`runs Axe on ${component.name} (${component.pageName})`, async () => {
```

Comment:
A description like "checks accessibility of _____" might be better since it's easier for someone who doesn't know the details of our tool names to understand.

Reply:
Makes sense! Updated.